amplivelist/amp-live-list.go

```diff
@@ -69,13 +69,27 @@ func createBlogEntry(text string, imagePath string) BlogItem {
 	return BlogItem{Text: text, Image: imagePath, Timestamp: currentTimestamp()}
 }
 
```

Comment:
The following three methods share 75% of the same code. Can you remove the duplication?